control/control.CASP.xml

```diff
@@ -314,44 +295,41 @@ Please visit us at http://www.suse.com/.
         <service_sfcb><label>CIM Server</label></service_sfcb>
         <roles_caption>
           <!-- TRANSLATORS: dialog caption -->
-          <label>System Role</label>
+          <label>CASP Roles</label>
         </roles_caption>
         <roles_text>
           <!-- TRANSLATORS: label in a dialog -->
-          <label>System Roles are predefined use cases which tailor the system
+          <label>Roles are predefined use cases which tailor the system
 for the selected scenario.</label>
         </roles_text>
         <roles_help>
           <!-- TRANSLATORS: dialog help -->
-          <label>&lt;p&gt;The system roles adjustments are in the range from package selection up 
-to disk partitioning. By choosing a system role, the system is 
-configured accordingly to match the use case of the role. The settings 
-defined by a role can be overridden in the next steps if necessary.&lt;/p&gt;</label>
+          <label>&lt;p&gt;The roles adjustments are in the range from package selection up
+to disk partitioning. By choosing a CASP role, the system is
+configured accordingly to match the use case of the role. &lt;/p&gt;</label>
         </roles_help>
-        <normal_role>
+        <dashboard_role>
+          <!-- TRANSLATORS: a label for a system role -->
+          <label>Admin Dashboard</label>
+        </dashboard_role>
+        <dashboard_role_description>
+          <label>• Default system with administration dashboard</label>
+        </dashboard_role_description>
+        <worker_role>
           <!-- TRANSLATORS: a label for a system role -->
-          <label>Default System</label>
-        </normal_role>
-        <normal_role_description>
-          <label>• GNOME environment, with Btrfs root (/) partition
-• Separate /home partition (XFS) for disks larger than 20GB</label>
-        </normal_role_description>
-        <kvm_host_role>
+          <label>Worker</label>
+        </worker_role>
+        <worker_role_description>
+          <label>• Default system registering as worker at the admin dashboard</label>
+        </worker_role_description>
+        <none_role>
           <!-- TRANSLATORS: a label for a system role -->
-          <label>KVM Virtualization Host</label>
-        </kvm_host_role>
-        <kvm_host_role_description>
-          <label>• Kernel-based hypervisor and tools
-• No separate /home partition</label>
-        </kvm_host_role_description>
-        <xen_host_role>
+          <label>None</label>
+        </none_role>
+        <none_role_description>
           <!-- TRANSLATORS: a label for a system role -->
-          <label>Xen Virtualization Host</label>
-        </xen_host_role>
-        <xen_host_role_description>
-          <label>• Bare metal hypervisor and tools
-• No separate /home partition</label>
-        </xen_host_role_description>
+          <label>• Default system, no services started by default</label>
```

Comment:
Here the "Default system" makes some sense. But I don't understand why someone would install a CASP with the None role. Why not install a plain SLES, or a JEOS? This needs to be explained better.